src/elements/kw-share-detail/kw-share-detail.html

```diff
@@ -352,7 +362,14 @@
                         <h2>Loading</h2>
                     </div>
                 </div>
-                <div id="share-container"></div>
+                <div class="content">
+                    <template is="dom-if" if="[[share.featured]]">
```

Comment:
Although this is only the share detail, which is only being loaded one at a time, but sure. I've made the update. I used a declared property, which is set when the share changes, rather than using `hidden$="[[!share.featured]]"` to prevent the badge flashing for a split-second while Polymer loads the share and computes whether this should be shown or not.